### ISSUE TITLE
[BITV] Modal next/previous buttons contrast possibly too low

### ISSUE THREAD

**Issue by PVince81** [nextcloud/nextcloud-vue#2954]
Steps:
1. Needs https://github.com/nextcloud/nextcloud-vue/pull/2953 (or click https://deploy-preview-2953--nextcloud-vue-components.netlify.app/#/Components/Modal)
2. Open styleguide
3. Scroll down and click on "Show modal with more properties"
4. See the arrow buttons on the left and right of the modal

<img width="700" alt="image" src="https://user-images.githubusercontent.com/277525/183092802-2cfcc1a9-9aaf-4300-88f1-dfdd4369b570.png">

Now this is with a white background from the styleguide + darkened overlay color.

Whatever the solution is, I'm not sure if it can work with all backgrounds ?

The focus effect looks fine though when tabbing:
<img width="154" alt="image" src="https://user-images.githubusercontent.com/277525/183093317-17d19da4-c445-4f34-907e-3c44f84b6bd5.png">

**Comment by PVince81** [comment]
@jancborchardt @nimishavijay

**Comment by jancborchardt** [comment]
Since the background is always made dark with that overlay, the icons should be white – then it works contrast-wise as well. :)

**Comment by PVince81** [comment]
@jancborchardt I assume white only when not hovered

**Comment by PVince81** [comment]
@Pytal can you adjust ? this will is the last item in https://github.com/nextcloud/nextcloud-vue/issues/2501 so we can then close that one too

**Comment by Pytal** [comment]
Fixed in https://github.com/nextcloud/nextcloud-vue/pull/3080
